### ISSUE TITLE
File upload

### ISSUE THREAD

**Issue by amellini** [tryAGI/Ollama#69]
There's a way to upload a document (in this case an Image) and add a request (Open webui allow this) i'd like to use this library to make an Image recognition.
It is possible?

**Comment by HavenDV** [comment]
Yes, use imagesAsBase64 parameter in the Chat.SendAsync or Message.Images property(and pass base64 to it)
